shlr/grub/fs/ext2.c

```diff
@@ -385,7 +385,11 @@ grub_ext2_read_block (grub_fshelp_node_t node, grub_disk_addr_t fileblock)
 
   if (grub_le_to_cpu32(inode->flags) & EXT4_EXTENTS_FLAG)
     {
-      char buf[EXT2_BLOCK_SIZE(data)];
+#ifndef _MSC_VER
+	  char buf[EXT2_BLOCK_SIZE (data)];
```

Comment:
dont do this. this will allocate too much mem in the stack and probably crash. just replace grub_malloc for malloc(